src/aiohabit/utils.py

```diff
@@ -133,6 +133,18 @@ def print_obj(obj):
     print(object2json(obj))
 
 
+def get_host_from_metadata(metadata, name):
```

Comment:
`name` is not used, this function always returns the first host, is it expected?

Reply:
good catch, fixed, filtering by name added